bin/shjs

```diff
@@ -1,12 +1,21 @@
 #!/usr/bin/env node
-require('../global');
 
-if (process.argv.length < 3) {
-  console.log('ShellJS: missing argument (script name)');
+function exit(msg) {
```

Comment:
nit: rename this function (it's too easy to confuse this with `shell.exit()`). Suggestion: `exitWithErrorMessage()`